docs/troubleshooting.md

```diff
@@ -68,7 +68,14 @@ This happens when gcsfuse is mounted with http1 client (default) and the applica
 
 ### Permission Denied error.
 
-Please refer [here](https://cloud.google.com/storage/docs/gcsfuse-mount#authenticate_by_using_a_service_account) to know more about permissions.(e.g.  **Issue**:mkdir: cannot create directory ‘gcs/test’: Permission denied. User can check specific errors by enabling logs with --log-severity=TRACE flags. **Solution**: Provide roles/storage.objectAdmin role on the bucket.)  <br/>
+Please refer [here](https://cloud.google.com/storage/docs/gcsfuse-mount#authenticate_by_using_a_service_account) to know more about permissions
+(e.g.  **Issue**:mkdir: cannot create directory ‘gcs/test’: Permission denied. User can check specific errors by enabling logs with --log-severity=TRACE flags.  
+**Solution** - depending upon the use-case, you can choose one of the following options.
```

Comment:
Both the solutions are a requirement, can we rephrase to something like:

**Issue:** `mkdir: cannot create directory ‘gcs/test’: Permission denied.` User can check specific errors by enabling logs with --log-severity=TRACE flags.

**Solution:** To resolve this issue, you need to ensure both of the following are in place:

1. **Grant necessary permissions to the service account:** Explicitly provide the service account with the roles/storage.objectAdmin role on the bucket. This allows the service account to create directories and write objects within the bucket.

2. **Configure VM scopes for Cloud Storage access:** If you are using GCP VM, set the VM's scope to storage-full or read-write to give the VM write-access to Cloud Storage buckets. This enables the VM to interact with Cloud Storage. To do this:
<insert steps>